jjb/dynamic/scale-ci-ms-rosa.yml

```diff
@@ -37,8 +37,14 @@
             sudo /usr/bin/yum -y install python3 python3-pip golang-bin make
             echo "INFO: Installing python requirements..."
             mkdir -p ${REMOTE_WORKSPACE}
-            [[ ! -d "${REMOTE_WORKSPACE}/osde2e-scale-wrapper" ]] && git clone https://github.com/cloud-bulldozer/osde2e-scale-wrapper.git ${REMOTE_WORKSPACE}/osde2e-scale-wrapper
-            sudo pip3 install -r ${REMOTE_WORKSPACE}/osde2e-scale-wrapper/requirements.txt
+            if [[ ! -d "${REMOTE_WORKSPACE}/perfscale-managed-services" ]] ; then
+              git clone https://github.com/cloud-bulldozer/perfscale-managed-services.git ${REMOTE_WORKSPACE}/perfscale-managed-services
```

Comment:
Thoughts on using the git plugin to clone the repo similar to https://github.com/cloud-bulldozer/scale-ci-pipeline/blob/master/jjb/dynamic/node-density-light.yml#L158?

Reply:
The scm step is executed on the host executing the job, and the clone Im executing is done on the jumping host.

Thats the reason I cannot use the scm plugin

Reply:
Makes sense.